tests/test_hooks.py

```
@@ -286,39 +286,122 @@ def tearDown(self):
-                    self.assertEqual(port, 9090)
-                    # The registry is now imported from metrics module, so it's the real registry
-                    from gunicorn_prometheus_exporter.metrics import (
-                        registry as metrics_registry,
-                    )
 
-                    self.assertEqual(registry, metrics_registry)
-                    mock_collector.assert_called_once_with(metrics_registry)
-                    mock_logger.debug.assert_called_once_with(
-                        "Successfully initialized MultiProcessCollector"
-                    )
+def test_start_single_attempt_other_error():
```

Pylint (3.3.8)

[refactor] Too many statements (107/50)

(R0915)